### DISCUSSION TITLE
dose KOR support large documents like 50-80pages pdf

### DISCUSSION THREAD

**Discussion by gouse95** [eyurtsev/kor#290] [Q&A]
while passing large context document it couldn't able to extract key info from PDF files can you help me out what approch will be good for larger pdf documents

**Comment by eyurtsev** [comment]
Here is some reference documentation. 

1) Split the document into chunks
2) Loop through each chunk and run extraction on it
3) Merge the results

https://python.langchain.com/docs/use_cases/extraction/how_to/handle_long_text/#brute-force-approach
https://eyurtsev.github.io/kor/document_extraction.html

**Reply by eyurtsev** [reply]
If you have a tool calling capable model you can use: https://github.com/langchain-ai/langchain-extract
